components/layout_2020/table/layout.rs

```diff
@@ -374,8 +374,7 @@ impl<'a> TableLayout<'a> {
         // >   * 100% minus the sum of the intrinsic percentage width of all prior columns in
         // >     the table (further left when direction is "ltr" (right for "rtl"))
         let mut total_intrinsic_percentage_width = 0.;
-        for column_index in 0..self.table.size.width {
-            let column_measure = &mut column_measures[column_index];
+        for column_measure in column_measures.iter_mut().take(self.table.size.width) {
```

Comment:
I think the `.take(self.table.size.width)` is not needed, `column_measures` shouldn't be longer than `self.table.size.width`.

Reply:
I think so too, it shouldn't have more than that, we should probably take it out